packages/fx-core/src/plugins/resource/bot/appStudio/appStudio.ts

```diff
@@ -152,6 +152,33 @@ export class AppStudio {
     return app;
   }
 
+  public static async getBotRegistration(
+    accessToken: string,
+    botId: string
+  ): Promise<IBotRegistration | undefined> {
+    const axiosInstance = AppStudio.newAxiosInstance(accessToken);
+
+    const getBotRegistrationResponse: AxiosResponse<any> | undefined = await RetryHandler.Retry(
```

Comment:
I think if you change 
```
AxiosResponse<any> 
```
to 
```
AxiosResponse<IBotRegistration>
```
then you can use those code
```
return getBotRegistrationResponse.data;
```
in line 176

Reply:
I agree with you. It may be slightly better. 
How about **updating this along with future PRs** as I don't want all PR checks to run again because of this subtle change.

Reply:
I agree with you - It may be slightly better. 
How about **updating this along with future PRs** as I don't want all PR checks to run again because of this subtle change.